docs/io/optional/logging_configuration.ipynb

```diff
@@ -0,0 +1,2226 @@
+{
```

Comment:
<p>Rewording of the first sentence, removing both commas, moving 'in functionality' for clarity, and changed 'functional argument' to 'parameter'.</p><p> </p><p>The <code>logging_level</code> parameter is similar in functionality to the <code>log_state</code> parameter that can be passed via the <code>run_tardis()</code> function.</p><p> </p><p>Remove the space before the colon. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1633/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>